docker-compose.yml

```diff
@@ -97,6 +97,9 @@ services:
       - redis
       - pender
       - alegre
+      - presto-server
+      - presto-image
+      - presto-audio
```

Comment:
Check API doesn't need to depend on these services directly, please remove these lines.